build.sbt

```diff
@@ -205,7 +205,17 @@ lazy val mtags = project
     buildInfoPackage := "scala.meta.internal.mtags",
     buildInfoKeys := Seq[BuildInfoKey](
       "scalaCompilerVersion" -> scalaVersion.value
-    )
+    ),
+    unmanagedSourceDirectories in Compile ++= {
```

Comment:
Is this change needed anymore?

Reply:
I think so, `AdaptedReporter` is present in 2.13.0 but it's gone in 2.13.1. I welcome alternative ideas :D